Sources/Core/JavaModelRenderer.swift

```diff
@@ -32,6 +33,18 @@ public struct JavaModelRenderer: JavaFileRenderer {
         } else {
             unknownPropertyLogging = nil
         }
+
+        if let uriType = JavaURIType(rawValue: params[.javaURIType] ?? "String") {
```

Comment:
Maybe we can use the enum case type here vs inlining a string literal?
`.string.rawValue` vs `"String"`?